### PR TITLE
Fix page url on shazbat.tv

### PR DESCRIPTION
**Pull request by duramato**

### CODE REVIEW

File: sickbeard/providers/shazbat.py [hunk 1]
```diff
@@ -76,7 +77,7 @@ def __init__(self, provider):
 
     def _getRSSData(self):
 
-        rss_url = self.provider.url + 'rss/recent?passkey=' + provider.passkey + '&fname=true'
+        rss_url = self.urls['base_url'] + 'rss/recent?passkey=' + provider.passkey + '&fname=true'
```
Comment (miigotu) [inline sickbeard/providers/shazbat.py]: `self.provider.urls['base_url']`

**Comment by duramato** [comment]
@miigotu done ,but why is it that way?

**Comment by miigotu** [comment]
Because it is not in the provider class, it is in the TVCache class for the provider. To access the provider attributes from the cache class in each provider you must go through `self.provider.`

**Comment by duramato** [comment]
oh i see thanks for the info
also @miigotu,the rebase its done :)

**Pull request merged**